e2e/interchaintestv8/relayer_test.go

```diff
@@ -1027,7 +1027,17 @@ func (s *RelayerTestSuite) ICS20TimeoutFromCosmosTimeoutTest(
 	s.Require().True(s.Run("Receive packets on Ethereum after timeout should fail", func() {
 		ics26Address := ethcommon.HexToAddress(s.contractAddresses.Ics26Router)
 		receipt, err := eth.BroadcastTx(ctx, s.EthRelayerSubmitter, 5_000_000, ics26Address, txBodyBz)
+		// The transaction itself can succeed to broadcast, but in that case, the receipt should show a failed transaction
+		if receipt == nil {
+			s.Require().Error(err)
+		} else {
+			s.Require().Equal(ethtypes.ReceiptStatusFailed, receipt.Status)
+		}
```

Comment:
Should this be a separate function or be used anywhere else? Do we know why this only happens here

Reply:
Not 100% sure why this happens, but we could potentially move this into the actual call itself to error. We do something similar for Cosmos broadcasts (where broadcasts often succeed, but the tx could be failed)?

Reply:
I think maybe we can move this in the function call and return an error in failed case?

Reply:
Yeah, agreed. Will change that